string pack
string node
---
bool result
